runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/OrchestratorRequest.java

```diff
@@ -0,0 +1,14 @@
+package org.corfudb.protocols.wireprotocol.orchestrator;
+
+import org.corfudb.format.Types.WorkflowType;
+import org.corfudb.protocols.wireprotocol.ICorfuPayload;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.corfudb.protocols.wireprotocol.ICorfuPayload'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)